erpnext/public/js/checkin_checkout_attendance.js

```diff
@@ -0,0 +1,39 @@
+frappe.provide('frappe.desktop');
+
+frappe.model.get_value("HR Settings", "HR", "checkin_check_out",
+	function(d) {
+		if(parseInt(d.checkin_check_out)){
+			frappe.call({
+				method: "erpnext.hr.doctype.attendance.attendance.validate_attendance",
+				callback: function(r) {
+					if (!r.message) {
+						show_popup();
+					}
+				}
+			});
+		}
+	});
+
+
+function show_popup() {
+	var dialog = new frappe.ui.Dialog({
```

Comment:
If user does not logged out till the next date, what will happen? May be check the server time every few minutes and if day changes, show the popup again.